app/views/school-administrator/teachers/SchoolAdminTeacherListView.vue

```diff
@@ -149,7 +149,7 @@ export default {
           v-if="administratedTeachers.length === 0"
           class="no-teachers"
         >
-          {{ $t('school_administrator.no_teachers') }}
+          <span v-html="$t('school_administrator.no_teachers')" />
```

Comment:
can you fix this warning by using eslint-disable-line with the rule that is causing this issue? We don't care about xss attack since it is not user supplied string so we just need to disable eslint on this line for that eslint rule